tests/views/test_collection_exercise.py

```diff
@@ -198,503 +142,260 @@ class File:
-                    ],
-                    "RU_REF": [],
-                    "SURVEY_ID": survey_id,
-                },
-                "file_name": "file",
-                "id": collection_instrument_id,
-                "surveyId": survey_id,
-                "type": "SEFT",
-            },
-            {
-                "classifiers": {
-                    "COLLECTION_EXERCISE": [
-                        collection_exercise_id,
-                    ],
-                    "RU_REF": [],
-                    "SURVEY_ID": survey_id,
-                },
-                "file_name": "file",
-                "id": collection_instrument_id_2,
-                "surveyId": survey_id,
-                "type": "SEFT",
-            },
-        ]
-        self.collection_exercise_ref_start_date = [
-            {
-                "id": collection_exercise_event_id,
-                "collectionExerciseId": collection_exercise_id,
-                "tag": "ref_period_start",
-                "timestamp": "2018-03-16T00:00:00.000Z",
-                "eventStatus": "PROCESSED",
-            }
-        ]
-        self.collection_exercise_ref_end_date = [
-            {
-                "id": collection_exercise_event_id,
-                "collectionExerciseId": collection_exercise_id,
-                "tag": "ref_period_end",
-                "timestamp": "2018-03-16T00:00:00.000Z",
-                "eventStatus": "PROCESSED",
-            }
-        ]
-        self.collection_exercise_ref_both_date = [
-            {
-                "id": collection_exercise_event_id,
-                "collectionExerciseId": collection_exercise_id,
-                "tag": "mps",
-                "timestamp": "2018-03-16T00:00:00.000Z",
-                "eventStatus": "PROCESSED",
-            },
-            {
-                "id": collection_exercise_event_id,
-                "collectionExerciseId": collection_exercise_id,
-                "tag": "ref_period_start",
-                "timestamp": "2018-03-16T00:00:00.000Z",
-                "eventStatus": "PROCESSED",
-            },
-            {
-                "id": collection_exercise_event_id,
-                "collectionExerciseId": collection_exercise_id,
-                "tag": "ref_period_end",
-                "timestamp": "2018-03-16T00:00:00.000Z",
-                "eventStatus": "PROCESSED",
+MOCK_GET_COLLECTION_EXERCISE_AND_SURVEY_DETAILS = (
+    "response_operations_ui.views.collection_exercise.get_collection_exercise_and_survey_details"
+)
+MOCK_BUILD_COLLECTION_INSTRUMENTS_DETAILS = (
+    "response_operations_ui.views.collection_exercise._build_collection_instruments_details"
+)
+MOCK_GET_SAMPLE_SUMMARY = "response_operations_ui.views.collection_exercise.get_sample_summary"
+MOCK_GET_COLLECTION_EXERCISE_EVENTS_BY_ID = (
+    "response_operations_ui.views.collection_exercise."
+    "collection_exercise_controllers."
+    "get_collection_exercise_events_by_id"
+)
+
+EQ_CI = {
```

Comment:
Eventually these should be fixtures in conftest, this is a step in the process of detangling, I don't want them to be attributes of a test class